### ISSUE TITLE
Image sequence input path?

### ISSUE THREAD

**Issue by twilson90** [mifi/editly#281]
I have a PNG sequence (0000.png - 9999.png) @ 30 FPS that I want to use as an input clip, is this possible?

**Comment by bkeepers** [comment]
@wisekaa03 definitely. You should be able to run `editly *.png` to stitch them together. I'm closing this to tidy up open issues, but let me know if that doesn't work for you.
